XamlControlsGallery/ControlPages/PageTransitionPage.xaml

```diff
@@ -16,7 +16,7 @@
                 <TextBlock Margin="0,0,0,8">Transition modes</TextBlock>
                 <RadioButton x:Name="DefaultRB" Content="Default" IsChecked="True" Checked="TransitionRadioButton_Checked" AutomationProperties.Name="Default NavigationTransitionInfo"/>
                 <RadioButton x:Name="EntranceRB" Content="Entrance" Checked="TransitionRadioButton_Checked" AutomationProperties.Name="EntranceNavigationTransitionInfo" />
-                <RadioButton x:Name="DrillRB" Content="Drill" Checked="TransitionRadioButton_Checked" AutomationProperties.Name="DrillNavigationTransitionInfo" />
+                <RadioButton x:Name="DrillRB" Content="Drill" Checked="TransitionRadioButton_Checked" AutomationProperties.Name="DrillInNavigationTransitionInfo" />
```

Comment:
```suggestion
                <RadioButton x:Name="DrillRB" Content="DrillIn" Checked="TransitionRadioButton_Checked" AutomationProperties.Name="DrillInNavigationTransitionInfo" />
```

I believe you also need to change the Content so that the code sample gets updated correctly.

![image](https://user-images.githubusercontent.com/13142665/70184368-8dec3c00-169c-11ea-936f-2b20e4cb1996.png)

Reply:
Yes you are right, the content should also have the right value